force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -183,14 +248,18 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<Integer> createAndPopulateList() {
         // Create a new list of integers
+        List<Integer> integerList = new List<Integer>();
 
         // Add the integers 1, 2, and 3 to the list
         // add 1
         // add 2
         // add 3
+        for (Integer x = 1; x <= 3; x++) {
+            integerList.add(x);
+        }
 
         // Return the populated list
-        return null; // Replace null with the variable you used to store the result
+        return integerList; // Replace null with the variable you used to store the result
```

Comment:
Good job. Senior dev tip. It is easier to use the List class constructor if you know all your values when you are writing. 

```
 public static List<Integer> createAndPopulateList() {
      return new List<Integer>{1, 2, 3};
    }
    ```